src/server/ssh_utils/__init__.py

```diff
@@ -6,13 +6,13 @@
 
 def get_fingerprint(public_key_filename):
     """
-    Return Fingerprint
+    Returns a key fingerprint
     """
     try:
-        fingerprint = check_output([
+        fingerprint = '\n'.join(check_output([
             'ssh-keygen',
```

Comment:
FAOR: Is there no OpenSSH pure python lib rather than calling system `ssh-keygen` ?

Reply:
https://www.paramiko.org/ might be the answer :)